doc/src/vpr/fasm.rst

```diff
@@ -0,0 +1,264 @@
+FPGA Assembly (FASM) Output Support
+===================================
+
+VPR can emit a FASM_ file to represent
```

Comment:
Looks like this may be outdated. I think genfasm is now it's own utility (which re-uses libvpr).

Should probably provide a general overview of how this works here:

- Perform implementation with VPR
- Run genfasm which loads the produced VPR output files (.net, .place, .route) and then generates the FASM file